lib/traceroute-lite.js

```diff
@@ -87,13 +87,16 @@ module.exports = Traceroute;
  * @param   {String} host An FQDN or IP address to trace.
  * @returns {Object} Returns the base traceroute class.
  */
-function Traceroute(host) {
+function Traceroute(host, maxNullHops) {
```

Comment:
I apologize for the delay in reviewing this PR. I'm happy to merge, but I'd like to use an `options` object instead of `maxNullHops` integer in the function signature.